mailpoet/lib/EmailEditor/Engine/SettingsController.php

```diff
@@ -113,10 +113,18 @@ public function getEmailStyles(): array {
         ],
       ],
       'colors' => [
-        'background' => '#000000',
+        'background' => '#ffffff',
       ],
       'typography' => [
       ],
+      // Value are only for purpose of displaying in the preview component in style sidebar
+      'elements' => [
```

Comment:
I assume these are temporary and will be replaced by some set values or defaults. Am I correct?

Reply:
Yes, my expectation here is to merge the email editor default values and values set for an email post.